Dockerfile

```
@@ -109,63 +82,54 @@ RUN set -eux ;\
         nano \
         procps  `# ps command` \
         gnupg2  `# TODO: not sure why gnupg2 is needed`  ;\
-    curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - ;\
-    /bin/bash -c 'source /etc/os-release && echo "deb http://apt.postgresql.org/pub/repos/apt/ ${VERSION_CODENAME:?}-pgdg main ${PG_MAJOR:?}" > /etc/apt/sources.list.d/pgdg.list' ;\
+    curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor -o /usr/share/keyrings/postgresql-keyring.gpg ;\
+    /bin/bash -c 'source /etc/os-release && echo "deb [signed-by=/usr/share/keyrings/postgresql-keyring.gpg] http://apt.postgresql.org/pub/repos/apt/ ${VERSION_CODENAME:?}-pgdg main ${PG_MAJOR:?}" > /etc/apt/sources.list.d/pgdg.list' ;\
     DEBIAN_FRONTEND=noninteractive apt-get update ;\
     DEBIAN_FRONTEND=noninteractive apt-get install  -y --no-install-recommends \
         aria2     `# multi-stream file downloader - used by download-osm` \
         graphviz  `# used by layer mapping graphs` \
-        sqlite3   `# mbtiles file manipulations`   \
-        gdal-bin  `# contains ogr2ogr` \
-        osmctools `# osmconvert and other OSM tools` \
-        osmosis   `# useful toolset - https://wiki.openstreetmap.org/wiki/Osmosis` \
-        postgresql-client-${PG_MAJOR:?}  `# psql` \
-        \
-        # imposm dependencies
-        libgeos-dev \
-        libleveldb-dev \
-        libprotobuf-dev \
+        sqlite3   `# mbtiles file manipulations` \
+        postgresql-client-${PG_MAJOR:?} \
+        postgresql-${PG_MAJOR:?}-postgis-3 \
+        osmium-tool \
+        osmctools `# contains osmconvert` \
+        libgeos-dev `# Imposm dependency` \
+        libleveldb-dev `# Imposm dependency` \
+        libprotobuf-dev `# Imposm dependency` \
```

```suggestion
        libprotobuf-dev `# Imposm dependency` \
        unzip \
```

debian trixie stopped install `unzip` during update. Missing `unzip` breaks the `download-fonts` in OMT.